website/blog/2023-11-30-automated-security-versus-the-security-mindset/item.md

```diff
@@ -0,0 +1,163 @@
+-   Lowering the number of false positives
+-   Updating software
+-   Helping with compliance
+-   Enhancing incident response and threat intelligence
+
+## Why automation is a threat to cybersecurity
+
+If automation is such a popular and necessary asset in the cybersecurity field, why can't we automate everything?
+
+_Let’s think: Could over-reliance on automated security testing ultimately prove detrimental to cybersecurity and threaten the safety of your systems?_
+
+To help avoid this, we need to acknowledge that automation can't:
+
+-   Keep security teams up to date with new standards, such as the NIST Cybersecurity Framework; the ISO/IEC 27001 standard for information security management; the CIS Critical Security Controls; the OSSTMM; the Web Application Security Consortium (WASC 2.0); or the finance standard of PCI Data Security Standards for the payment card industry
+-   Adjust your internal security policies and practices to all the nuances of relevant industry, country or regulations such as NIST SP 800-52; The California Consumer Privacy Act: the Canadian PIPEDA; the EU’s GDPR; or HIPAA’s personal health data legislation
+-   Rapidly respond to every new CVE or every item that makes an appearance in the SANS Top 25, or the most common vulnerabilities listed in the OWASP lists
+-   Ensure that your own internal cybersecurity protocols and policies are enforced
+
+_But what else?_
+
+The first point to remember is that automated solutions can only reliably alert and respond to the threats to your network, services, databases, APIs, and applications that they've been configured to detect. This configuration is limited to the settings available in the particular software. Auto security testing tools are only as good as the rules that human security engineers have given them. They still must be configured and employed accurately.
+
+And, your own company’s internal business logic must be factored in. This is where pentesters (who may, of course, rely on some automated tools to help them identify some vulnerabilities across your network) can delve deep on specific vulnerabilities and apply your company’s custom business logic and data breach implications. Resultant summary reports must explain the business, financial, reputational, data, and user implications of likely breaches, investigations and penalties.
+
+Also, malicious hackers can use automated security techniques just as much as defenders to find potential security flaws in an organization’s network. They use novel attacks inspired by vulnerabilities that automated tools are unable to detect at all, by exploiting mistakes made by users that automation by itself can't solve. Examples include social engineering attacks that can begin with an innocuous looking email, or an SMS or email phishing scam. Given that over 80% of bad actors gain illegitimate entry using social engineering attacks, it is obvious that company-wide staff training is an excellent deployment of resources.
+
```

Comment:
... which you then repeat here as well.  

Suggestion: "Inspired by vulnerabilities that automated tools are unable to detect at all, they exploit mistakes made by users and developers that automation cannot solve by itself."